lib/tasks/time_entries_updater.rake

```diff
@@ -0,0 +1,7 @@
+desc 'Fetches time entries every 10 minutes and updates the employees'
+
+task time_entries_updater: :environment do
+  EmployeeService.new.update_all_entries
+end
+
+
```

Comment:
After the merging of this we should probably add it to heroku time synchronisation